api/src/graphql/users.sdl.js

```diff
@@ -15,14 +15,14 @@ export const schema = gql`
 
   input CreateUserInput {
     email: String!
-    issuer: String!
+    # issuer: String!
```

Comment:
What was happening here, was this causing problems?

Reply:
Yes, The error  was Field "issuer" of required type "String!" was not provided.
The Schema you have created does not use this field.